uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-request-record-actions.scss

```diff
@@ -0,0 +1,30 @@
+leave-request-record-actions {
+  display: block;
+
+  .leave-request-record-actions__button {
+    display: block;
+    float: left;
+    margin-left: 0;
```

Comment:
why are these styles necessary at all? `btn`s are `inline-block`, there should be any need for floating. I removed all these styles and they look fine in my local (and it allows you to get rid of the `.clearfix` wrapper

Reply:
When I turn Float of, an extra gap appears. This is what I tried to solve with "btn-toolbar". It appears because of `inline-block`. We can of course try to apply `:after, :before { content: '' }`.

Reply:
> When I turn Float off, an extra gap appears. 

please attach a screenshot, don't always wait for me to ask more details

Reply:
I finally refactored the code, so please ignore my comment about floats.

Reply:
![ddd](https://user-images.githubusercontent.com/3973243/37781958-b10447b6-2de9-11e8-8fa9-b5950bd3776b.gif)

Reply:
Let me please explain why we should use `float`. If we don't use floats, but use `inline-block` instead, then a gap will appear as you see on the GIF above. I attempted to try a lot of different solutions, but only the one with `float` works in **all** implementations of the component and is pixel perfect.